doc/manual/manual.tex

```diff
@@ -1138,6 +1138,46 @@ \subsection{Compositional fields}
 temperature set of equations. Play with them!
 
 
+\subsection{Constitutive laws}
+
+Equation (\ref{eq:stokes-1}) describes buoyancy-driven flow in an isotropic
+fluid where strain rate is related to stress by a scalar field, $\eta$. For
+some material models it is useful to generalize this relationship for anisotropic
+materials, or other exotic constitutive laws. For these cases \aspect{} can
+optionally include a generalized, fourth-order tensor field as a material model
+state variable which changes equation (\ref{eq:stokes-1}) to
+\begin{align}
+  \label{eq:stokes-1-anisotropic}
+  -\nabla \cdot \left[2\eta \left(C \varepsilon(\mathbf u)
+                                  - \frac{1}{3}(tr(C \varepsilon(\mathbf u)))\mathbf 1\right)
+                \right] + \nabla p &=
+  \rho \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$}
+\end{align}
+and the shear heating term in equation (\ref{eq:temperature}) to
+\begin{align}
+  \label {eq:temperature-anisotropic}
+  \dots
+  \notag
+  \\
+  + 2 \eta
+  \left(C \varepsilon(\mathbf u) - \frac{1}{3}(tr(C \varepsilon(\mathbf u)))\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  \\
+  \dots
+  \notag
+\end{align}
+where $C = C_{ijkl}$ is defined by the material model. This functionality can be
```

Comment:
Add a sentence here such as this:

`For physical reasons, $C$ needs to be a symmetric rank-4 tensor, i.e., when multiplied by a symmetric (strain rate) tensor of rank 2, it needs to return another symmetric tensor of rank 2. In mathematical terms, this means that $C_{ijkl}=C_{jikl}=C_{ijlk}=C_{jilk}$. Energy considerations also require that $C$ is positive definite, i.e., for any $\varepsilon\neq 0$, the scalar $\varepsilon : (C \varepsilon)$ must be positive.`